apps/storybook.namekit.io/stories/Namekit/DisplayedPrice.stories.tsx

```diff
@@ -40,654 +86,201 @@ export default meta;
-      value: 1n,
-    },
-    symbol: (
-      <span>
-        {getCurrencySymbology(Currency.Eth, CurrencySymbology.Acronym)}
-      </span>
-    ),
-  },
-};
-export const UsdcMaxDisplayPriceWithAcronym: Story = {
-  args: {
-    price: {
-      currency: Currency.Usdc,
-      value: 1000000000000000000n,
-    },
-    symbol: (
-      <span>
-        {getCurrencySymbology(Currency.Eth, CurrencySymbology.Acronym)}
-      </span>
-    ),
-  },
-};
-export const DaiMinDisplayPriceWithAcronym: Story = {
-  args: {
-    price: {
-      currency: Currency.Dai,
-      value: 1n,
-    },
-    symbol: (
-      <span>
-        {getCurrencySymbology(Currency.Eth, CurrencySymbology.Acronym)}
-      </span>
-    ),
-  },
-};
-export const DaiMaxDisplayPriceWithAcronym: Story = {
-  args: {
-    price: {
-      currency: Currency.Dai,
-      value: 10000000000000000000000000n,
-    },
-    symbol: (
-      <span>
-        {getCurrencySymbology(Currency.Eth, CurrencySymbology.Acronym)}
-      </span>
-    ),
-  },
-};
-export const WethMinDisplayPriceWithAcronym: Story = {
-  args: {
-    price: {
-      currency: Currency.Weth,
-      value: 1n,
-    },
-    symbol: (
-      <span>
-        {getCurrencySymbology(Currency.Eth, CurrencySymbology.Acronym)}
-      </span>
-    ),
-  },
-};
-export const WethMaxDisplayPriceWithAcronym: Story = {
-  args: {
-    price: {
-      currency: Currency.Weth,
-      value: 1000000000000000000000000n,
-    },
-    symbol: (
-      <span>
-        {getCurrencySymbology(Currency.Eth, CurrencySymbology.Acronym)}
-      </span>
-    ),
-  },
-};
-export const EthMinDisplayPriceWithSymbol: Story = {
-  args: {
-    price: {
-      currency: Currency.Eth,
-      value: 100000000000000n,
-    },
+    price: numberAsPrice(0.001, Currency.Eth),
     symbol: (
       <CurrencySymbol
         currency={Currency.Eth}
-        describeCurrencyInTooltip={true}
+        symbology={CurrencySymbology.Icon}
       />
     ),
   },
 };
-export const EthMaxDisplayPriceWithSymbol: Story = {
+export const EthMaxDisplayPriceWithCustomCurrencyIcon: Story = {
   args: {
-    price: {
-      currency: Currency.Eth,
-      value: 10000000000000000000000000n,
-    },
+    price: numberAsPrice(1000000, Currency.Eth),
     symbol: (
       <CurrencySymbol
         currency={Currency.Eth}
-        describeCurrencyInTooltip={true}
+        symbology={CurrencySymbology.Icon}
       />
     ),
   },
 };
-export const UsdMinDisplayPriceWithSymbol: Story = {
+export const UsdMinDisplayPriceWithCustomCurrencyIcon: Story = {
   args: {
-    price: {
-      currency: Currency.Usd,
-      value: 1n,
-    },
+    price: numberAsPrice(0.01, Currency.Usd),
     symbol: (
       <CurrencySymbol
         currency={Currency.Usd}
-        describeCurrencyInTooltip={true}
+        symbology={CurrencySymbology.Icon}
       />
     ),
   },
 };
-export const UsdMaxDisplayPriceWithSymbol: Story = {
+export const UsdMaxDisplayPriceWithCustomCurrencyIcon: Story = {
```

Comment:
We should rename these "max" or "min" price stories.

Instead of "max" we should use the word "overflow".

Instead of "min" we should use the word "underflow".

It's true that each `Currency` has a min or max display price. But those are single fixed values. There can be an infinite number of `Price` values that are higher than the max display price for a given `Currency. It doesn't matter which of those values we choose in the story, whenever there is an "overflow" we display the max display price.

Does this make sense? Please feel welcome to ask me questions.

Reply:
It does! Do you think I should rename this story, thus?

Reply:
It does!